### PR TITLE
[GCI-142] Removing unused import

### PR DESCRIPTION
**Pull request by Devanshk9**
## Description of what I changed
Removed unused import 'org.openmrs.customdatatype.InvalidCustomValueException'on' 
AS requested by sonar

## Issue I worked on
I worked on a sonar issue- https://sonar.openmrs.org/project/issues?id=org.openmrs%3Aopenmrs&open=AWcHAZiJ8fYkPidqGdVn&resolved=false&severities=MINOR

see https://issues.openmrs.org/browse/GCI-142

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by MrUbaidUllahZahid** [comment]
Thank you @Devanshk9 for your code and changes I have the resolve the problems.

File: api/src/main/java/org/openmrs/GlobalProperty.java [hunk 1]
```diff
@@ -15,7 +15,6 @@
 import org.openmrs.customdatatype.CustomDatatype;
 import org.openmrs.customdatatype.CustomDatatypeUtil;
 import org.openmrs.customdatatype.CustomValueDescriptor;
-import org.openmrs.customdatatype.InvalidCustomValueException;
```
Comment (f4ww4z) [inline api/src/main/java/org/openmrs/GlobalProperty.java]: Is this the only unused import in the entire codebase?

Comment (Devanshk9) [reply api/src/main/java/org/openmrs/GlobalProperty.java]: @f4ww4z according to sonar yeah only one is there

Comment (f4ww4z) [reply api/src/main/java/org/openmrs/GlobalProperty.java]: I said in the entire codebase. Please merge this PR with #3107 then close this.

**Comment by dkayiwa** [comment]
@Devanshk9 can you make the commit message reflect the actual change instead of the generic Update FormResource?

**Pull request closed without merging**